microprofile/cdi/src/main/java/module-info.java

```diff
@@ -14,12 +14,23 @@
  * limitations under the License.
  */
 
+import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.HelidonFlavor;
+
 /**
  * CDI implementation enhancements for Helidon MP.
  *
  * @see jakarta.enterprise.context
  */
+@Feature(value = "CDI",
+        description = "Jakarta CDI implementation",
+        in = HelidonFlavor.MP,
+        path = "CDI"
+)
 module io.helidon.microprofile.cdi {
+    requires static io.helidon.common.features.api;
```

Comment:
We cannot have `requires static` here, as Helidon flavor is used at runtime

Reply:
Fixed